tests/admin_views/tests.py

```diff
@@ -551,6 +551,18 @@ def test_has_related_field_in_list_display(self):
         response.context['cl'].list_display = ['id', 'name', 'state_id']
         self.assertIs(response.context['cl'].has_related_field_in_list_display(), False)
 
+    def test_has_related_field_in_list_display_o2o(self):
```

Comment:
It's fine to include this test but since it's already passing without the fix, it's better in a separate commit.